packages/slice-machine/src/pages/changes.tsx

```diff
@@ -84,7 +96,11 @@ const Changes: React.FunctionComponent = () => {
         // Update last sync value in local storage
         pushChangesSuccess();
 
-        toast.success("All slices and types have been pushed");
+        // TODO: hide this toast for users in experiment eligible version (DT-2287)
+        setIsToastOpen(true);
```

Comment:
I'll refactor this to use useToast hook when [PR with some fixes in `editor-ui`](https://github.com/prismicio/editor/pull/1193) is merged,
This will make this component and managing toast cleaner.